lib/components/Dropdown/Dropdown.tsx

```diff
@@ -62,14 +62,11 @@ const NamedDropdown = forwardRef<HTMLSelectElement, DropdownProps>(
 
     return (
       <Field {...restProps} ref={ref} secondaryMessage={null}>
-        {(
-          { className, paddingLeft, paddingRight, ...fieldProps },
-          fieldRef,
-        ) => (
+        {({ className, paddingX, ...fieldProps }, fieldRef) => (
           <Fragment>
             <Box
               component="select"
-              paddingLeft={paddingLeft}
+              paddingLeft={paddingX}
```

Comment:
```suggestion
              paddingX={paddingX}
```

Reply:
No, this was actually deliberate. It's applying the horizontal padding, but only on the left side.

Reply:
Weird...

Reply:
It's because it's applying the right padding to a different element.